vtm-android-example/build.gradle

```diff
@@ -1,6 +1,7 @@
 apply plugin: 'com.android.application'
 
 dependencies {
+    compile project(':vtm')
```

Comment:
That's strange..
VTM on Android Studio / Gradle releases should work `as is` or else we should fix it.
Also Travis has not complained so far about dependencies.
Have you disabled `transitive` in your Gradle properties somehow?